cron/weekly/weekly-report.js

```diff
@@ -1,12 +1,12 @@
-const app = require('../index');
```

Comment:
You could rename to `weekly-slack-report.js` similarly to the `monthly-slack-report.js`. Also, as it's in directory `weekly`, could just be named `slack-report.js`.